src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -12185,8 +12186,20 @@ public void TestAnalyzerFilteringBasedOnSeverity(DiagnosticSeverity defaultSever
             }
             else
             {
-                Assert.Contains("warning AD0001: Analyzer 'Microsoft.CodeAnalysis.CommonDiagnosticAnalyzers+NamedTypeAnalyzerWithConfigurableEnabledByDefault' threw an exception of type 'System.NotImplementedException'",
+                Assert.Contains(GetWarningMessage(currentCulture),
                     output, StringComparison.Ordinal);
+
+
+                string GetWarningMessage(CultureInfo culture)
```

Comment:
>string GetWarningMessage(CultureInfo culture) [](start = 16, length = 45)

This doesn't look like the right thing to do. We are passing ```/preferreduilang:en``` switch. All messages are supposed to follow the requested culture. The test failure is likely indicating a real bug in the product.